cmd/notation/internal/display/metadata/tree/tree.go

```diff
@@ -25,35 +25,35 @@ const (
 	subTreePrefixLast  = "    "
 )
 
-// represents a Node in a tree
-type Node struct {
+// represents a node in a tree
+type node struct {
 	Value    string
-	Children []*Node
+	Children []*node
 }
 
-// creates a new Node with the given value
-func New(value string) *Node {
-	return &Node{Value: value}
+// creates a new node with the given value
+func new(value string) *node {
```

Comment:
`new` is a golang syntax keyword. Consider `newNode`.

Reply:
Good idea.